### PR TITLE
add event dispatchers when lists curation & creation happen

### PR DESCRIPTION
**Pull request by iisa**
Dev notes:

i did not use analytics package as petabox has different api. (send_event instead of sendEvent)
also, lots of stuff happening in component, hard to parse, so opted for keeping analytics responsibility in petabox.
Testing: Open Demo, open dev console, watch console logs fire

### CODE REVIEW

File: demo/app-root.ts [hunk 1]
```diff
@@ -9,7 +9,19 @@ export class AppRoot extends LitElement {
...
+          @memberRemoved=${(e: CustomEvent) =>
+            console.log('memberRemoved', e.detail)}
+          @listCreateOpen=${(e: CustomEvent) =>
+            console.log('listCreateOpen', e.detail)}
+          @userItemListDataReceived=${(e: CustomEvent) =>
+            console.log('userItemListDataReceived', e.detail)}
+          @closeDropdown=${() => console.log('closeDropdown')}
+          @selectDropdown=${(e: CustomEvent) =>
+            console.log('selectDropdown', e.detail)}
+        ></ia-item-user-lists>
```
Comment (iisa) [inline demo/app-root.ts:24]: hard to test without mock data source, but adding it here for example.

File: src/item-user-lists.ts [hunk 2]
```diff
@@ -151,10 +156,18 @@ export class ItemUserLists extends LitElement {
...
   }
 
   private async removeMember(listId: string, memberId: string): Promise<void> {
     await this.userListsService?.removeMemberFromList(listId, memberId);
+    this.dispatchEvent(
+      new CustomEvent('removeMember', {
+        detail: { listId, itemId: this.itemId },
+      })
+    );
   }
```
Comment (latonv) [inline src/item-user-lists.ts:171]: Q: do we want these to be named `memberAdded` and `memberRemoved` for consistency with the above?

**Review by latonv — APPROVED** [review]
LGTM, just had a small naming consistency Q.

File: src/item-user-lists.ts (hunk 2 above)
Comment (iisa) [reply src/item-user-lists.ts:171]: keeping it to match function name and action for the events calls - to keep close to this code.
when received in petabox, it will change to `memberAdded` as that is what we want in analytics.
